src/openrct2/scripting/bindings/network/ScPlayerGroup.cpp

```diff
@@ -56,7 +56,7 @@ namespace OpenRCT2::Scripting
         auto result = s.substr(sizeof("PERMISSION_") - 1);
         for (auto& c : result)
         {
-            c = std::tolower(c);
+            c = std::tolower(static_cast<unsigned char>(c));
         }
         return result;
```

Comment:
Could just return ```String::ToLower(result)``` maybe?

Reply:
There is no `String::ToLower` currently, but the getter function here could make use of `String::ToUpper` instead of the loop. Changed that one anyway, left the rest as-is for a future refactor PR.